src/dmd/dsymbolsem.d

```diff
@@ -2199,12 +2199,15 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             }
             else if (auto se = ns.exp.toStringExp())
                 ns.ident = identFromSE(se);
+            // Empty Tuple
+            else if (ns.exp.isTypeExp() && ns.exp.isTypeExp().type.toBasetype().isTypeTuple())
+            {
+            }
```

Comment:
So empty tuple defaults to `TypeTuple` instead of `TupleExp`, and I guess that's why the special case ?

Reply:
Yes and No. Tuple as returned by `__traits` is a `TupleExp`, an empty `AliasSeq` is a type tuple. Why I have no Idea.

Reply:
Looking at the diff, I think it's because it's assigned to an `alias`. Try adding `immutable ns2 = AliasSeq!();` and using that as well.

Reply:
done.